### ISSUE TITLE
Can't use v2: 'Markdown' was not found in 'react-marked-renderer'

### ISSUE THREAD

**Issue by nyg** [mlaursen/react-marked-renderer#196]
Hello,

I have the following error when trying to use `react-marked-renderer@2.0.0` in a very simple project:

```
Failed to compile.

Attempted import error: 'Markdown' is not exported from 'react-marked-renderer' (imported as 'Markdown').
ERROR in ./src/index.js 16:33-41
export 'Markdown' (imported as 'Markdown') was not found in 'react-marked-renderer' (module has no exports)

webpack compiled with 1 error
```

My `index.js` is the following:

```js
import React from 'react';
import ReactDOM from 'react-dom/client';
import { Markdown } from 'react-marked-renderer';

const markdown = `# Heading 1

This is some text that will be rendered as a paragraph.

Markdown defaults to the github-flavored markdown.
`;

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
   <React.StrictMode>
      <Markdown markdown={markdown} />
   </React.StrictMode>
);
```

<details>
  <summary>package.json</summary>

  ```json
  {
    "name": "markdown-test",
    "version": "0.1.0",
    "private": true,
    "dependencies": {
      "github-markdown-css": "5.2.0",
      "react": "18.2.0",
      "react-dom": "18.2.0",
      "react-marked-renderer": "2.0.0",
      "react-scripts": "5.0.1"
    },
    "scripts": {
      "start": "react-scripts start",
      "build": "react-scripts build",
      "eject": "react-scripts eject"
    },
    "browserslist": {
      "production": [
        ">0.2%",
        "not dead",
        "not op_mini all"
      ],
      "development": [
        "last 1 chrome version",
        "last 1 firefox version",
        "last 1 safari version"
      ]
    }
  }
  ```

</details>

With v1.1.2 it works without any issues. I use Node 18.15.0 & npm 9.5.0.

Am I doing something wrong?

Thanks,
nyg

**Comment by nyg** [comment]
Any idea @mlaursen? Let me know if I'm doing something wrong.

**Comment by mlaursen** [comment]
@nyg Sorry for the delay. I forgot to add the `exports` field to the `package.json` after enabling esm which was causing this issue. I published `2.0.1` which should fix this issue for CRA.

**Comment by nyg** [comment]
Great, thanks  a lot! The issue is gone. No worries for the delay.
